app/src/pages/explorer/sample/sample.html

```diff
@@ -3,59 +3,63 @@
     <spinner :loading="loading" :text="'Loading sample page'" />
   </div>
   <div v-else>
-    <div v-if="header" data-test="header">
-      <h1 class="visualize_header-h1 article_title u_centralize_text">Sample ID: {{ this.$route.params.label.toUpperCase() }}</h1>
-      <div class="u_centralize_text utility-color utility-line-height-sm">
-        <span class="utility-navfonticon">{{ header.sample_label.toUpperCase() }}</span>
-        <span class="utility-navfonticon">{{ header.process_label.toUpperCase() }}</span>
-        <div>DOI: {{ header.DOI }}</div>
-      </div>
-    </div>
-    <div style="max-width: 99%;">
+    <div style="max-width: 99%">
       <div class="md-layout md-gutter" :class="`md-alignment-top-center`">
         <div class="md-layout-item md-size-50">
+          <div v-if="header" data-test="header">
+            <h1 class="md-headline article_title u_centralize_text">
+              {{`${header.sample_label.toUpperCase()} ${header.process_label.toUpperCase()}`}}
+            </h1>
+          </div>
+          <h2 v-if="sampleAuthor" class="sample-page_header">Metadata:</h2>

```

Comment:
The v-if here only applies to the h2, not to the DOI in line 18

Reply:
my bad 😅 fixed on https://github.com/Duke-MatSci/materialsmine/pull/221/commits/d50c814c55ce31eefe0884d285227d07e0976aaf